test/compat/index.test.js

```diff
@@ -59,3 +61,137 @@ test("has a working `quoteAll` function", (t) => {
     t.is(typeof output, "string");
```

Comment:
With the addition of the `fast-check`-based tests the existing tests may be removed as they're no redundant at best or useless at worst (depending on the system where they're run).

Reply:
Resolved with 3471de0b34128e7c3809ba129b8d55097a19d90f